aiohttp/test_utils.py

```diff
@@ -413,7 +413,7 @@ def unittest_run_loop(func, *args, **kwargs):
     """
 
     @functools.wraps(func, *args, **kwargs)
-    def new_func(self):
+    def new_func(self, *args, **kwargs):
```

Comment:
I fear now there is a collision and it's a breaking change: previously args and kwargs were taken from unittest_run_loop arguments, now they get overridden by new_func arguments.